<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>index.utf8.md</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Danang Crysnanto</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="resume.html">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:danangcrysnanto@gmail.com">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/danangcrysnanto">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/crdanang">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="section" class="section level3">
<h3></h3>
</div>
<div id="section-1" class="section level3">
<h3></h3>
</div>
<div id="section-2" class="section level3">
<h3></h3>
</div>
<div id="section-3" class="section level3">
<h3></h3>
<p><a href="https://www.ethz.ch/en.html"><img src="figs/eth_logo.png" width="40%" height="40%" align="right"></a></p>
</div>
<div id="doctoral-student-at-eth-zurich-switzerland" class="section level3">
<h3>Doctoral Student at ETH Zurich Switzerland</h3>
<p><br></br> <br></br> <font size="3">I am a third year PhD student at <a href="http://www.ag.ethz.ch/">Animal Genomics</a> ETH Zurich Switzerland.</p>
<p>I have expertise in the computational methods to analyse genomic big data. I have 5 years experience in using data science stacks e.g., <em>R</em>, <em>Tidyverse</em>, <em>Python</em>, <em>Jupyter</em> for genomic analysis.</p>
<p>I am currently working to integrate millions of bovine genetic variants into a single comprehensive graph-based reference representation. <a href="https://www.biorxiv.org/content/10.1101/2019.12.20.882423v1">My recent work</a> showed that using reference graphs I implemented reduce the genomic sequencing mapping error by more than 30% compared to the current practices that represent reference genome as a linear string.</p>
<p>I am highly experienced in designing <a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0462-x">bioinformatics pipelines</a> for variant discovery in large-scale population level (scalable to thousands) of whole genome sequencing data. I mostly using <em>Snakemake</em> for pipelines design and management.</p>
<p>Previously I studied <a href="http://qgen.bio.ed.ac.uk/">MSc in Quantitative Genetics and Genome Analysis</a> at the University of Edinburgh UK, where I received two awards for my master’s project:</p>
<ol style="list-style-type: decimal">
<li>The Douglas Falconer Prize by Institute of Evolutionary of Biology Edinburgh UK for the best <em>Master’s Thesis</em>.</li>
<li><a href="http://www.genetics.org.uk/medals-and-prizes/sir-kenneth-mather-memorial-prize/sir-kenneth-mather-memorial-prize-2016-17/">Sir Kenneth Mather Memorial Prize</a> by the UK Genetics Society for an outstanding work in Population Genomics.</li>
</ol>
<p>I am a huge supporter for <em>reproducible</em> and <em>open</em> sciences in computational research. I will make efforts to make all <em>papers</em> I published (including <em>codes</em>), <em>posters</em>, and <em>presentations</em> I present in the conferences to be publicly available.</p>
<p></font> <br></br></p>
</div>
<div id="publications" class="section level3">
<h3><strong>Publications</strong></h3>
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<p><strong>Danang Crysnanto</strong>, Hubert Pausch. Bovine breed-specific augmented reference graphs facilitate accurate sequence read mapping and unbiased variant discovery. <a href="https://www.biorxiv.org/content/10.1101/2019.12.20.882423v3"><em>bioRxiv Preprint</em></a> from 20<sup>th</sup> December 2019. <a href="https://github.com/danangcrysnanto/bovine-graphs-mapping">Github repository</a>.</p>
<p><strong>Danang Crysnanto</strong>, Christine Wurmser, Hubert Pausch. 2018. <a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0462-x">Accurate sequence variant genotyping in cattle using genome graphs</a>. Published in: <em>Genetics Selection Evolution</em>. <a href="https://www.biorxiv.org/content/10.1101/460345v2"><em>bioRxiv Preprint</em></a> from 4<sup>th</sup> November 2018. <a href="https://github.com/danangcrysnanto/Graph-genotyping-paper-pipelines">Github repository</a>.</p>
<p><strong>Danang Crysnanto</strong>, Darren Obbard. 2018. <a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-019-1425-0">Widespread gene duplication and adaptive evolution in the RNA-interference pathways of the Drosophila obscura group</a>. Published in: <em>BMC Evolutionary Biology</em>. <a href="https://www.biorxiv.org/content/10.1101/429894v3"><em>bioRxiv Preprint</em></a> from 29<sup>th</sup> September 2018. <a href="https://github.com/danangcrysnanto/RNAi-duplication">Github repository</a>.</p>
<p>Bhati M, Kadri NK, <strong>Crysnanto D</strong> , Pausch H. Assessing genomic diversity and signatures of selection in Original Braunvieh cattle using whole-genome sequencing data. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6446-y"><em>BMC Genomics</em></a> from 16<sup>th</sup> July 2019.</p>
<p><br></br></p>
</div>
<div id="conferences-and-talks" class="section level3">
<h3><strong>Conferences and Talks </strong></h3>
<p>Plant and Animal Genomes Conference (PAG). San Diego USA. Mapping sequencing reads to bovine genome graphs. 11-15 Jan 2020. <em>Selected Oral Speaker</em>. <a href="https://doi.org/10.6084/m9.figshare.11494185.v1">Presentation slides</a> and <a href="https://doi.org/10.6084/m9.figshare.11494185.v1">Poster presentation</a>.</p>
<p>Genome Informatics and Cambridge Livestock Genomics Meeting. Wellcome Genome Campus Cambridge UK. Evaluation of genome graph-based variant genotyping in cattle. 21-22 Sept 2018. <em>Selected Oral Presenter, Long talk</em>. <a href="https://doi.org/10.6084/m9.figshare.11454906.v1">Presentation slides</a>.</p>
<p>52nd Population Genetics. Oxford UK. Widespread gene duplications in <em>Drosophila</em> immune system. 3-6 Jan 2019. <em>Invited Award Winner Speaker</em>. <a href="https://doi.org/10.6084/m9.figshare.11454900.v1">Presentation slides</a>.</p>
<p><br></br></p>
</div>
<div id="contact" class="section level3">
<h3><strong>Contact </strong></h3>
<p><i class="far fa-envelope"></i> <a href="mailto:danangcrysnanto@gmail.com">danangcrysnanto@gmail.com</a> <br> <i class="fab fa-twitter"></i> <a href="https://twitter.com/crdanang"><span class="citation">@crdanang</span></a></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
