###
###
###
###
<a href="https://www.ethz.ch/en.html"><img src="figs/eth_logo.png" width="40%" height="40%" align="right"></a>  

### Doctoral Student at ETH Zurich Switzerland  
<br></br>
<br></br>
<font size="3">I am a third year PhD student at [Animal Genomics](http://www.ag.ethz.ch/) ETH Zurich Switzerland. 


I have expertise in the computational methods to analyse genomic big data. I have 5 years experience in using data science stacks e.g., *R*, *Tidyverse*, *Python*, *Jupyter* for genomic analysis. 

I am currently working to integrate millions of bovine genetic variants into a single comprehensive graph-based reference representation. [My recent work](https://www.biorxiv.org/content/10.1101/2019.12.20.882423v1) showed that using reference graphs I implemented reduce the genomic sequencing mapping error by more than 30% compared to the current practices that represent reference genome as a linear string. 


I am highly experienced in designing [bioinformatics pipelines](https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0462-x) for variant discovery in large-scale population level (scalable to thousands) of whole genome sequencing data. I mostly using *Snakemake* for pipelines design and management.

Previously I studied [MSc in Quantitative Genetics and Genome Analysis](http://qgen.bio.ed.ac.uk/) at the University of Edinburgh UK, where I received two awards for my master's project:  

(1)  The Douglas Falconer Prize by Institute of Evolutionary of Biology Edinburgh UK for the best *Master's Thesis*.
(2)  [Sir Kenneth Mather Memorial Prize](http://www.genetics.org.uk/medals-and-prizes/sir-kenneth-mather-memorial-prize/sir-kenneth-mather-memorial-prize-2016-17/) by the UK Genetics Society for an outstanding work in Population Genomics.


I am a huge supporter for *reproducible* and *open* sciences in computational research. I will make efforts to make all *papers* I published (including *codes*), *posters*, and *presentations* I present in the conferences to be publicly available. 

</font>
<br></br>

### **Publications** 
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

**Danang Crysnanto**, Hubert Pausch. Bovine breed-specific augmented reference graphs facilitate accurate sequence read mapping and unbiased variant discovery. *Genome Biology In press*. bioRxiv Preprint from 20<sup>th</sup> December 2019. [Github repository](https://github.com/danangcrysnanto/bovine-graphs-mapping).


**Danang Crysnanto**, Christine Wurmser, Hubert Pausch. 2018. [Accurate sequence variant genotyping in cattle using genome graphs](https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0462-x). Published in: *Genetics Selection Evolution*.  bioRxiv Preprint from 4<sup>th</sup> November 2018. [Github repository](https://github.com/danangcrysnanto/Graph-genotyping-paper-pipelines).


**Danang Crysnanto**, Darren Obbard. 2018. [Widespread gene duplication and adaptive evolution in the RNA-interference pathways of the Drosophila obscura group](https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-019-1425-0). Published in: *BMC Evolutionary Biology*. bioRxiv Preprint from 29<sup>th</sup> September 2018. 
[Github repository](https://github.com/danangcrysnanto/RNAi-duplication).


Hiltpold M, Niu G, Kadri NK, **Crysnanto D**, Fang Z-H, Spengeler M, et al. (2020) [Activation of cryptic splicing in bovine WDR19 is associated with reduced semen quality and male fertility](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008804&rev=2) PLoS Genet 16(5) from 14<sup>th</sup> May 2020.


Bhati M, Kadri NK, **Crysnanto D** ,  Pausch H. Assessing genomic diversity and signatures of selection in Original Braunvieh cattle using whole-genome sequencing data. [*BMC Genomics*](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6446-y) from 16<sup>th</sup> July 2019.

Adéla Nosková, Christine Wurmser, **Danang Crysnanto** et. al. [Deletion of porcine BOLL causes defective acrosomes and subfertility in Yorkshire boars](https://www.biorxiv.org/content/10.1101/2020.05.05.074724v1.abstract). bioRxiv from 7<sup>th</sup> May 2020.

Zih-Hua Fang, Adéla Nosková, **Danang Crysnanto** et. al. [A 63-bp insertion in exon 2 of the porcine KIF21A gene is associated with arthrogryposis multiplex congenita](https://www.biorxiv.org/content/10.1101/2020.04.09.033761v1.abstract). bioRxiv from 10<sup>th</sup> April 2020. 

<br></br>


### **Conferences and Talks **

Plant and Animal Genomes Conference (PAG). San Diego USA. Mapping sequencing reads to bovine genome graphs. 11-15 Jan 2020. *Selected Oral Speaker*. [Presentation slides](https://doi.org/10.6084/m9.figshare.11494185.v1) and 
[Poster presentation](https://doi.org/10.6084/m9.figshare.11494185.v1). 

Genome Informatics and Cambridge Livestock Genomics Meeting. Wellcome Genome Campus Cambridge UK. Evaluation of genome graph-based variant genotyping in cattle. 21-22 Sept 2018. *Selected Oral Presenter, Long talk*. [Presentation slides](https://doi.org/10.6084/m9.figshare.11454906.v1).

52nd Population Genetics. Oxford UK. Widespread gene duplications in *Drosophila* immune system. 3-6 Jan 2019. *Invited Award Winner Speaker*. [Presentation slides](https://doi.org/10.6084/m9.figshare.11454900.v1).

<br></br>

### **Contact **      

<i class="far fa-envelope"></i>    <a href="mailto:danangcrysnanto@gmail.com">danangcrysnanto@gmail.com</a> <br>
<i class="fab fa-twitter"></i>     <a href="https://twitter.com/crdanang">@crdanang</a> 
