---
title: "Research Experience"
---
<br></br>
  
# {.tabset}

## Current  

### Genome graph for variant genotyping in cattle  
<font size="3.5">


Linear reference genome is an oversimplification as it fails to account for the segregated variants in the populations, resulting in overreporting the alleles in the reference, widely known as reference allele bias.  

A natural data structure that can represents these variants jointly is genome-graph. 
In this work, I showed that using cattle WGS (the first of its kind in livestock population) genome-graph genotyper is comparable, and even has better quality metrics (e.g genotype concordance, mendelian accuracy) than the current standard variant-calling tools that are strictly limited to the linear reference genome.


#### **Working paper**  
Danang Crysnanto, Christine Wurmser, Hubert Pausch. 2018. ["Accurate sequence variant genotyping in cattle using genome graphs"](https://www.biorxiv.org/content/early/2018/11/04/460345). *bioRxiv* from 4th November 2018.
</font>

## Past  
### The evolution of RNAi gene duplicates in *Drosophila*  
<font size="3.5">


Genes involved in RNA inteference pathway are prevasively duplicated in *D obscura* group.   
I analysed the 35 genomes across species in *Drosophilidae* and showed that the duplication are adaptive and occurred independently across *Drosophila* lineages, implying that many selective agents may be responsible.  


#### **Preprint**  
Danang Crysnanto, Darren Obbard. 2018. ["Widespread gene duplication and adaptive evolution in the RNA-interference pathways of the Drosophila obscura group"](https://www.biorxiv.org/content/early/2018/09/29/429894). *bioRxiv* from 29th September 2018.
</font>


